Apps/WebClient/src/NewClientApp/src/components/site/FeedbackComponent.vue

```diff
@@ -0,0 +1,253 @@
+                            We won't be able to respond to your message unless
+                            you have a verified email address in your profile.
+                        </template>
+                    </v-alert>
+                    <v-alert v-if="hasFailed" type="error" variant="text">
+                        <template #text>
+                            If the problem persists please send your feedback to
+                            <a href="mailto:HealthGateway@gov.bc.ca">
+                                HealthGateway@gov.bc.ca
+                            </a>
+                        </template>
+                    </v-alert>
+                </v-card-text>
+                <v-card-actions class="pa-4">
+                    <v-spacer />
+                    <template v-if="!isSuccess && !hasSubmitted">
+                        <HgButtonComponent
+                            data-testid="noNeedBtn"
+                            variant="link"
+                            text="Cancel"
+                            @click="hideDialog"
+                        />
+                        <HgButtonComponent
+                            data-testid="sendFeedbackMessageBtn"
+                            :disabled="isValid !== true"
+                            :loading="isLoading"
+                            text="Send Message"
+                            type="submit"
+                        />
+                    </template>
+                    <template v-else-if="isSuccessWithoutEmail">
+                        <HgButtonComponent
+                            data-testid="noNeedBtn"
+                            variant="link"
+                            text="No Need!"
+                            @click="hideDialog"
+                        />
+                        <HgButtonComponent
+                            data-testid="updateMyEmailButton"
+                            variant="primary"
+                            text="Update my email"
+                            to="/profile"
+                            @click="hideDialog"
+                        />
+                    </template>
+                    <!-- Single button States (centered) -->
+                    <template v-else>
+                        <HgButtonComponent
+                            v-if="isSuccessWithEmail"
+                            data-testid="hasEmailResetFeedbackBtn"
+                            text="Got it!"
+                            variant="primary"
+                            :loading="isLoading"
+                            @click="hideDialog"
+                        />
+                        <HgButtonComponent
+                            v-if="hasFailed"
+                            data-testid="tryAgainBtn"
+                            text="Try Again"
+                            variant="primary"
+                            :loading="isLoading"
+                            :disabled="isValid !== true"
```

Comment:
```suggestion
                            :loading="isLoading"
                            :disabled="isLoading || isValid !== true"
```

When both apply, the disabled prop (unfortunately?) overrides the disabled styling applied by the loading prop.

Reply:
Sigh, cool thanks for that, I assumed the loading would take priority.